contents/docs/integrate/_snippets/install-ruby.mdx

```diff
@@ -7,7 +7,7 @@ gem "posthog-ruby"
 In your app, set your API key **before** making any calls. If setting a custom `host`, make sure to include the protocol (e.g. `https://`).
 
 ```ruby
-require 'posthog-ruby'
+require 'posthog'
```

Comment:
> Huh, what did we change that causes this? The class -> module change?

@rafaeelaudibert the only other thing I can think is rubocop made me rename `posthog-ruby` file to `posthog_ruby`

but the class -> module change I guess is the most likely candidate